src/test/java/com/ably/kafka/connect/validators/ChannelNameValidatorTest.java

```diff
@@ -0,0 +1,26 @@
+package com.ably.kafka.connect.validators;
+
+import org.apache.kafka.common.config.ConfigException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ChannelNameValidatorTest {
```

Comment:
We have tests for `:` and `[` but there are some other invalid characters left from the regex pattern (namely whitespace and `,`), I think we should test them too as they feel as much important as the ones that are already tested :wink:

Reply:
b511308bd68a1556a594d80e032e6b3c121e4268